.github/workflows/release.yml

```diff
@@ -89,14 +89,14 @@ jobs:
   #     - uses: ./.github/actions/setup-sfdx
   #     - name: Promote SFDX package to production
   #       run: |
-  #         echo "${{ secrets.SFDX_AUTH_PACKAGE_JWT_KEY }}" > server.key
+  #         echo "${{ secrets.SFDX_AUTH_JWT_KEY }}" > ${{ vars.SFDX_AUTH_JWT_KEY_FILE }}
```

Comment:
The job is still commented out at this point, but I'm getting it ready

Reply:
You're reserving this for KIT-3072 I guess?

Reply:
Aye, although at this point it's probably going to be trivial